generics/services/sunbird.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * name : sunbird.js
+ * author : Vishnudas
+ * Date : 07-Mar-2022
+ * Description : All Sunbird learner related api call.
+ */
+
+//dependencies
+
+
+const request = require('request');
+const sunbirdBaseUrl = process.env.SUNBIRD_SERVICE_URL;
```

Comment:
@VISHNUDAS-tunerlabs check SUNBIRD_SERVICE_URL and USER_SERVICE_URL is same ? if both are same remove the new env variable SUNBIRD_SERVICE_URL